File: Python/Kode_Program/Aksesoris.py
from Petshop import Petshop  # Mengimpor kelas Petshop dari file terpisah

class Aksesoris(Petshop):
    # Konstruktor untuk inisialisasi objek dengan atribut default
    def __init__(self, jenis="", ukuran="", warna=""):
        self.Jenis = jenis
        self.Ukuran = ukuran
        self.Warna = warna

    # Mengatur jenis aksesoris
    def Set_Jenis(self, jenis):
        self.Jenis = jenis

    # Mengatur ukuran aksesoris
    def Set_Ukuran(self, ukuran):
        self.Ukuran = ukuran

    # Mengatur warna aksesoris
    def Set_Warna(self, warna):
        self.Warna = warna

    # Mengambil jenis aksesoris
    def Get_Jenis(self):
        return self.Jenis
    
    # Mengambil ukuran aksesoris
    def Get_Ukuran(self):
        return self.Ukuran
    
    # Mengambil warna aksesoris
    def Get_Warna(self):
        return self.Warna

   

File: PHP/Kode_Program/Aksesoris.php
<?php
include_once 'Petshop.php'; // Pastikan Petshop di-include
// Class Baju merupakan turunan dari class Aksesoris
class Aksesoris extends Petshop {
    // Properties
    private $jenis, $bahan, $warna;

    // Constructor
    public function __construct() {
        
        $this->jenis = "";
        $this->bahan = "";
        $this->warna = "";
    }

   
    // Setter
    public function Set_Jenis($jenis) { $this->jenis = $jenis; }
    public function Set_Bahan($bahan) { $this->bahan = $bahan; }
    public function Set_Warna($warna) { $this->warna = $warna; }

    // Getter
    public function Get_Jenis() { return $this->jenis; }
    public function Get_Bahan() { return $this->bahan; }
    public function Get_Warna() { return $this->warna; }
}
?>
